<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>991</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Ajouter client</string>
    </attribute>
    <widget class="QLabel" name="label_cin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prenom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_adresse">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomAjout">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="cinAjout">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenomAjout">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="telAjout">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>380</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuller</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bouton_ajout">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bouton_image">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD IMAGE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>tunis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bizerte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kairouan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gafsa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>beja</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>siliana</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jendouba</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nabeul</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ariana</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tozeur</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2">
    <attribute name="title">
     <string>Afficher liste clients</string>
    </attribute>
    <widget class="QLabel" name="label_nom_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bouton_supprimer">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>320</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bouton_PDF">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>460</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Export PDF</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_client">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>11</y>
       <width>411</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>251</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Modifier Client</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cinModif">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomModif">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenomModif">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresseModif">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numModif">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bouton_modifier">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bouton_validerMod">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioButton_croissant">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par nom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_decroissant">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par prenom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_date">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>320</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par date d'inscription</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>359</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Chercher un client</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>statistiques</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>641</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
